distro/scenarios/theNewWorld/culture.properties

```diff
@@ -339,7 +339,6 @@
 1673,1354,FARME
 696,350,HUNTE
 2023,1812,FARME
-2272,1736,HUNTE

```

Comment:
Found out the reason: There is an NPE because of the negative value, will change to positive number.